index.html

```diff
@@ -77,64 +90,8 @@ <h2>Main Program</h2>
     </div>
   </section>
 
-  <section class="speakers">
+  <section id="speakers" class="speakers">
     <h3 class="featured-title">Featured Speakers</h3>
```

Comment:
- [ ] Kindly make sure to add a separator after the `Featured Speaker` Headline, so your website looks identical to the provided design

|Your website|CC design|
|-|-|
|![Screenshot_20230318_000715](https://user-images.githubusercontent.com/25227022/226067132-43706c5c-0d90-44f7-90b9-eeedaef86683.png)|![Screenshot_20230318_000701](https://user-images.githubusercontent.com/25227022/226067118-63616990-5df3-4378-b1db-912a990bce2c.png)|